<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>773</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Singapore Airlines - In-flight Catering Manager</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="flightManagerGUI">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Flight Manager</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Flight_CreateButton">
            <property name="text">
             <string>Create New Flight</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Flight_SearchButton">
            <property name="text">
             <string>Search Flights</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Flight_RefreshButton">
            <property name="text">
             <string>Refresh Table</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Flight_UpdateButton">
            <property name="text">
             <string>Update Flights</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scanLeftoversButton">
            <property name="text">
             <string>Scan Leftovers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Flight_ViewMealScheduleButton">
            <property name="text">
             <string>View Meals Served on Selected Flight</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="Flight_Table">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>400</height>
           </size>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>135</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mealScheduleManagerGUI">
       <attribute name="title">
        <string>Meal Schedule</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="MealSchedule_CreateButton">
            <property name="text">
             <string>Create New Meal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MealSchedule_SearchButton">
            <property name="text">
             <string>Search Meals</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MealSchedule_RefreshButton">
            <property name="text">
             <string>Refresh Table</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MealSchedule_UpdateButton">
            <property name="text">
             <string>Update Meal Schedule</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MealSchedule_ViewMealOptionButton">
            <property name="text">
             <string>View Options for Selected Meal</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="MealSchedule_Table">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>135</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mealOptionManagerGUI">
       <attribute name="title">
        <string>Meal Options</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="MealOption_CreateButton">
            <property name="text">
             <string>Create New Meal Option</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MealOption_SearchButton">
            <property name="text">
             <string>Search Meal Options</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MealOption_RefreshButton">
            <property name="text">
             <string>Refresh Table</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MealOption_UpdateButton">
            <property name="text">
             <string>Update Meal Options</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="MealOption_Table">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>135</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
